master/reference/calicoctl/resources/ippool.md

```diff
@@ -70,6 +70,17 @@ Increasing the block size from the default (e.g., using `24` for IPv4 to give 25
 
 Reducing the block size from the default (e.g., using `28` for IPv4 to give 16 addresses per block) means more blocks per host and therefore potentially more routes. This can be beneficial if it allows the blocks to be more fairly distributed amongst the hosts.
 
+#### Node Selector
+
+{% include {{page.version}}/selectors.md %}
+
+For details on configuring IP pool node selectors on your deployment, please read the
+[Assigning IP addresses based on toplogy guide.]({{site.baseurl}}/{{page.version}}/usage/assigning-ip-addresses-topology).
+
+> **Note**: The pool's `disabled` field takes higher precedence than `nodeSelector`
+> i.e. {{site.prodname}} IPAM will not allocate any IPs from the pool if disabled even if it selects the node that it is on.
```

Comment:
For "i.e." and "e.g.", you always enclose them in commas. So you just need a comma before and after i.e. here